api/server/router/system/system_routes.go

```diff
@@ -25,7 +25,12 @@ func optionsHandler(ctx context.Context, w http.ResponseWriter, r *http.Request,
 	return nil
 }
 
-func pingHandler(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
+func (s *systemRouter) pingHandler(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
+	if s.featureBuildKit != nil && *s.featureBuildKit {
```

Comment:
```Go
if s.featureBuildKit != nil {
    s := "false"
    if *s.featureBuildKit {
        s = "true"
    }
    w.Header().Set("Feature-BuildKit-Enabled", s)
}
```

Reply:
This can be simplified even further : 

```go
if s.featureBuildkit != nil {
    w.Header().Set("Feature-BuildKit-Enabled", fmt.Sprintf("%v", *s.featureBuildkit))
}
```

Reply:
@tiborvass done